omnibus/files/private-chef-ctl-commands/chef-server-ctl

```diff
@@ -216,6 +216,17 @@ EOM
         "/admin/ping?wt=json"
       end
     end
+
+    # Override reconfigure to skip license checking
```

Comment:
Why is this needed? As an open source project, the license acceptance code should not be triggered. If that's the case, this is working around a bug and this is not the right fix.